src/main/java/com/checkout/common/ChallengeIndicator.java

```diff
@@ -8,19 +8,9 @@ public enum ChallengeIndicator {
     CHALLENGE_REQUESTED,
     @SerializedName("challenge_requested_mandate")
     CHALLENGE_REQUESTED_MANDATE,
-    @SerializedName("data_share")
```

Comment:
Hey. Alex from Wise lurking here 👋 
Wondering if this is considered a breaking change?

Reply:
I checked it, I made this mistake adding this string here. But I can leave it

Reply:
I leave the correct values https://api-reference.checkout.com/#operation/requestAPaymentOrPayout!path=0/3ds/2/challenge_indicator&t=request

Reply:
Please @ogaragatyi resolve this conversation

Reply:
Weird think - I resolved my comment but not the whole thread. See this kind of behavior from Github for the first time